validation/vertical_mixing_closures/column_windy_convection.jl

```diff
@@ -30,7 +30,7 @@ coriolis = FPlane(f=f₀)
 b_bcs = FieldBoundaryConditions(top = FluxBoundaryCondition(Jᵇ))
 u_bcs = FieldBoundaryConditions(top = FluxBoundaryCondition(τˣ))
 #closures_to_run = [catke, ri_based, convective_adjustment]
-closures_to_run = [catke] #, ri_based, convective_adjustment]
+closures_to_run = [catke] #, ri_based] #, convective_adjustment]
```

Comment:
?

Reply:
Why this line change with extra comment in the comment..? But not that important tbh

Reply:
it's two separate changes, one to compare only with catke + ri_based, and then a second to run just catke. I removed it